ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -201,6 +201,12 @@ export default class AlertTable extends React.Component {
       : null;
     const repoModel = new RepositoryModel(repo);
 
+    const sortedAlerts = orderBy(
```

Comment:
Could we rename this to reflect it contains both filtered **and** sorted alerts?